packages/nextjs/tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "jsx": "preserve",
     "incremental": true,
     "paths": {
-      "~~/*": ["./*"]
+      "~~/*": ["./*"],
+      "@root/*": ["../../*"]
```

Comment:
I still get this on my nextjs console:

![image](https://user-images.githubusercontent.com/2486142/224970191-854acd4e-5136-41e0-9628-25fd6c1f7f27.png)

I ran `yarn install` and restarted nextjs. Am I missing something?

Reply:
Yes !! encountered the same problem for both, don't know for some reason it was working but I am sure i am missing some configuration, working on it